include/openssl/ocsp.h

```diff
@@ -164,9 +164,26 @@ OPENSSL_EXPORT int OCSP_resp_find_status(OCSP_BASICRESP *bs, OCSP_CERTID *id,
                                          ASN1_GENERALIZEDTIME **thisupd,
                                          ASN1_GENERALIZEDTIME **nextupd);
 
-// OCSP_basic_verify verifies a basic response message. Returns 1 if the
-// response is valid, 0 if the signature cannot be verified, or -1 on fatal
-// errors such as malloc failure.
+// OCSP_check_validity checks the validity of |thisUpdate| and |nextUpdate|
+// fields from an |OCSP_SINGLERESP|.
+//
+// Note: It is possible that the request will take a few  seconds to process
+// and/or the time won't be totally accurate. Therefore, to avoid rejecting
```

Comment:
```suggestion
// Note: It is possible that the request will take a few seconds to process
// and/or the local system time isn't exactly the same as the OCSP responder's time. Therefore, to avoid rejecting
```